src/client/Microsoft.Identity.Client/AppConfig/TraceTelemetryConfig.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using Microsoft.Identity.Client.TelemetryCore.Internal;
+using Microsoft.Identity.Client.Utils;
+using Microsoft.Identity.Json;
+using Microsoft.Identity.Json.Linq;
+
+namespace Microsoft.Identity.Client
+{
+    /// <summary>
+    /// A simple <see cref="ITelemetryConfig"/> implementation that writes data using System.Diagnostics.Trace.
+    /// </summary>
+    /// <remarks>This API is experimental and it may change in future versions of the library without an major version increment</remarks>
+    public class TraceTelemetryConfig : ITelemetryConfig
```

Comment:
Is this a new file? Or was it just moved?

Reply:
Never mind, I think GH shows it as new because of spacing. CodeFlow shows me the changes.

Reply:
May be because I removed it initially, then readded back (since it's public).